.github/workflows/ensure-backported.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Ensure changes are migrated to Vue3
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened, synchronize, labeled]
+    branches:
+      - main
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  commit-message-check:
+    if: github.event.pull_request.draft == false
```

Comment:
Should we also check, that it is not a PR from a bot?

Reply:
I would say we can exclude dependabot - because that is useless for that branch.
All other bots need manual review anyways and probably should also be ported.